api/app/services/spree/webhooks/subscribers/handle_request.rb

```diff
@@ -5,7 +5,7 @@ module Webhooks
     module Subscribers
       class HandleRequest
         def initialize(body:, event_name:, subscriber:)
-          @body = body
+          @body = JSON.parse(body)
```

Comment:
@sebastian-palma so this is the payload body right? not the response?

Reply:
The only purpose of this change is to avoid `ActiveJob::SerializationError`s as it raises an unsupported type error for the specific case of the order serializer (ActiveJob::SerializationError: Unsupported argument type: Spree::Money). We could convert those attributes to string but that would work only for OrderSerializer, if any other serializer has an unsupported type that can not automatically be serialized it'll result in the same error @damianlegawiec.